src/main/java/application/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package application.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.antlr.v4.runtime.misc.NotNull;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities
```suggestion
@Getter
@Setter
@Table(name...
```